README.md

```diff
@@ -19,12 +19,148 @@ Python packages with our recommended tooling set up and ready to go.
 > package template (e.g., [`scikit-hep`](https://github.com/scikit-hep/cookie)),
 > we recommend using their template instead of this one.
 
+## Tutorial
+
+Some quick instructions for using our template are below.
+We also have a pedagogical [tutorial](./tutorial.md) that has been given in a couple of workshops geared towards researchers at UCL.
+
+## Using our template
+
+If you have [uv] installed, you can use our template with the following one-liner:
+
+```sh
+uvx cookiecutter gh:ucl-arc/python-tooling --checkout latest
+```
+
+You can also [install cookiecutter].
+Do this if you don't use [uv], or if you're likely to want to use cookiecutter again.
+Here's the way we install it:
+
+```sh
+uv tool install cookiecutter
+```
+
+Then you'll need to run cookiecutter with our template:
+
+```sh
+cookiecutter gh:ucl-arc/python-tooling --checkout latest
+```
+
+When [cookiecutter] runs, it will ask you a series of questions to configure your project.
+Type the answer or hit return without typing anything to use the default option (shown in parenthesis).
+At the end, it will print some more follow-up information in the terminal for things like creating a remote repository and making a website for your package.
+
+Here's an example run through
+
+<details><summary>Click to expand...</summary>
+
+```sh
+uvx cookiecutter gh:ucl-arc/python-tooling --checkout latest
+  [1/14] Given name(s) of package author (Eva Lu): Grace
+  [2/14] Family name(s) of package author (Ator): Hopper
+  [3/14] Email address for package author - will be part of package metadata
+(grace.hopper@ucl.ac.uk):
+  [4/14] Name of project - may contain spaces (Python Template): Example Research Software Project
+  [5/14] 'Slugified' project name for use in URLs - dash-case recommended
+(example-research-software-project):
+  [6/14] Name for Python package - snake_case recommended
+(example_research_software_project): ersp
+  [7/14] Short description of the project (A cookiecutter package with UCL ARC
+recommendations.): An example project.
+  [8/14] Initialise project directory as a Git repository? [y/n] (y): y
+  [9/14] Automatically deploy HTML docs to GitHub Pages on pushes to main? [y/n]
+(y): y
+  [10/14] GitHub user or organization name which will be owner of repository
+(grace-hopper): UCL-ARC
+  [11/14] Minimum Python version supported by package
+    1 - 3.11
+    2 - 3.12
+    3 - 3.13
+    Choose from [1/2/3] (1):
+  [12/14] Maximum Python version supported by package
+    1 - 3.13
+    2 - 3.12
+    3 - 3.11
+    Choose from [1/2/3] (1):
+  [13/14] Which open-source license to release package under
+    1 - MIT
+    2 - BSD-3
+    3 - GPL-3.0
+    Choose from [1/2/3] (1): 1
+  [14/14] Organisation(s) to acknowledge for funding of project (optional) ():
```

Comment:
Do we want this? It's yet another thing to maintain that won't be very DRY, in addition to #432

Reply:
Maybe not... I mean it _is_ quite self-explanatory.

Reply:
I'm 👍 to removing this

Reply:
I would also be in favour of removing to avoid having to keep up to date